website_remove_powered_by_odoo/README.rst

```diff
@@ -6,51 +6,8 @@
 Website Remove Powered by Odoo
 ==============================
 
-* This module extends the functionality of website and allows you to remove Powered by Odoo from the website.
```

Comment:
I think the version you want to merge is older than the one in the repo.
Can you recheck it?